src/main/java/hardcorequesting/PlayerDeathEventListener.java

```diff
@@ -21,4 +30,50 @@ public void onPlayerDeath(LivingDeathEvent event) {
         }
     }
 
+    @SubscribeEvent(priority = EventPriority.HIGHEST)
+    public void onPlayerDropItemsOnDeath(PlayerDropsEvent event)
```

Comment:
No otherwise it would still drop a book, or be put in a gravestone, or handled by whatever other mod that does things with drops